MIGRATION.md

```diff
@@ -1428,17 +1498,77 @@ To migrate your `Tooltip` component to v4, run the following [codemod](https://g
 If you want to do it manually, you must transform your existing code as follows:
 
 ```tsx
+import { Tooltip } from '@contentful/forma-36-react-components';
+
 <Tooltip content="content of the Tooltip" containerElement="div" place="left">
   <TextLink>Hover me</TextLink>.
-</Tooltip>
+</Tooltip>;
 ```
 
 Into:
 
 ```tsx
+import { Tooltip } from '@contentful/f36-components';
+
 <Tooltip content="content of the Tooltip" as="div" placement="left">
   <TextLink>Hover me</TextLink>.
-</Tooltip>
+</Tooltip>;
+```
+
+## Workbench
+
+Workbench in version 4 got moved into a separate package, but since we would like to deprecate this component, we don't include it in the main build. In order to use this component you need to install it separately:
```

Comment:
```suggestion
Workbench in version 4 got moved into a separate package, but since we would like to deprecate this component, we don't include it in the main package. In order to use this component, you need to install it from a separate package:
```